Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -270,6 +273,66 @@ void MenuBar::AddTableColumnsMenu(QMenu* view_menu)
   }
 }
 
+void MenuBar::AddShowPlatformsMenu(QMenu* view_menu)
+{
+  static const QMap<QString, bool*> platform_menu{
+    {tr("Show Wii"), &SConfig::GetInstance().m_ListWii},
+    {tr("Show GameCube"), &SConfig::GetInstance().m_ListGC},
+    {tr("Show WAD"), &SConfig::GetInstance().m_ListWad},
+    {tr("Show ELF/DOL"), &SConfig::GetInstance().m_ListElfDol}};
+
+  QActionGroup* platform_group = new QActionGroup(this);
+  QMenu* plat_menu = view_menu->addMenu(tr("Show Platforms"));
```

Comment:
``platform_menu``. Rename the current ``platform_menu`` for something more descriptive to do that (See above).